src/js/timeseries.js

```diff
@@ -63,7 +63,16 @@ function getChange(data, options) {
 		return;
 	}
 
-	const oldest = getPrimaryMetric(data[0], options);
+	let oldestIndex;
+
+	for (let i = 0; i < data.length; i++) {
+		if (getPrimaryMetric(data[i], options) > 0) {
+			oldestIndex = i;
+			break;
+		}
+	}
+
+	const oldest = getPrimaryMetric(data[oldestIndex], options);
```

Comment:
Before this line can you add

```js
if (oldestIndex === undefined) {
  return;
}
```

If all values are 0 we won't have an index and it'll probably show `NaN` or something. This should hide the change entirely.